src/services/expenses.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+let expenses = [];
+
+function getExpenses() {
```

Comment:
```suggestion
function resetExpenses() {
```
mind consistent naming